src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -116,8 +116,7 @@
     // use by Google-written clients.
     func (c *{@view.clientTypeName}) SetGoogleClientInfo(name, version string) {
         goVersion := strings.Replace(runtime.Version(), " ", "_", -1)
-        v := fmt.Sprintf("%s/%s %s gax/%s go/%s", name, version, gapicNameVersion, gax.Version, goVersion)
-        c.metadata = metadata.Pairs("x-goog-api-client", v)
+        c.xGoogHeader = fmt.Sprintf("gl-go/%s %s/%s gax/%s", goVersion, name, version, gax.Version)
```

Comment:
Let's include `grpc/UNKNOWN` as the last element of this string. We're doing that in Java when we can't determine it, and I think will @jskeet do the same with .NET.